.github/workflows/sdk-size-updates.yml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches:
       - develop
-
+    paths-ignore:
+      - 'README.md'
```

Comment:
In that case, we don't need that anymore

Reply:
If, by some reason, only the Readme has been updated, we don't really need to rerun it either, right?

Reply:
By using [skip ci], no CI is triggered for any commit, no mater what was changed

Reply:
Many other files should not trigger this workflow but I think isn't worth it to include all of them there